README.md

```diff
@@ -87,15 +87,11 @@ The format is a gzipped JSON `genesis.json` file, with either:
 
 Run the following command from the `validation` folder to run the Go validation checks, for only the chain you added (replace the chain name or ID accordingly):
 ```
-go test -run=/OP-Sepolia
+go test -run=TestValidation/OP-Sepolia
 ```
 or
 ```
-go test -run=/11155420
-```
-You can even focus on a particular test and chain combination:
-```
-go test -run=TestGasPriceOracleParams/11155420
+go test -run=TestValidation//11155420
```

Comment:
It looks like there's an extra `/` in this command

Reply:
[d83482b](https://github.com/ethereum-optimism/superchain-registry/pull/350/commits/d83482be63a7e2b3208a08e6765e41a8b24ba28c)